redaxo/src/addons/phpmailer/lib/mailer.php

```diff
@@ -56,19 +56,44 @@ public function send()
             if ($this->log) {
                 $this->log();
```

Comment:
Muss es hier nicht `mail_archive()` sein?

Reply:
heißt jetzt archive() ... und $this->archive